webapp/src/projects.tsx

```diff
@@ -104,6 +104,14 @@ export class Projects extends auth.Component<ISettingsProps, ProjectsState> {
         applyCodeCardAction(this.props.parent, "projects", scr, action);
     }
 
+    async syncCloudProjects() {
+        await cloud.syncAsync();
```

Comment:
does this need `{ direction: "down" }` ?

Reply:
Default is to sync both directions, which I think is the more appropriate behavior when the sync is initiated by the user.